src/Model/TransportModel/tsp1ssm1.f90

```diff
@@ -111,6 +114,11 @@ subroutine ssm_cr(ssmobj, name_model, inunit, iout, fmi, eqnsclfac, &
     !    package has access to the corresponding dependent variable type
     ssmobj%depvartype = depvartype
     !
+    ! -- Give package access to the shared heat transport variables assigned in MST
+    if (present(gwecommon)) then
```

Comment:
Repeat comment here about "gwe" being in a general ssm package.

Reply:
Agreed, gwe-related code should not appear in the generalized transport modules. References to gwe-related code removed from `tsp1ssm1.f90` with 62ef0ba